forest/benchmarking/compilation.py

```diff
@@ -57,6 +57,17 @@ def _RY(angle, q):
     return p
 
 
+def _RX(angle, q):
+    """
+    A RX in terms of RX(+-pi/2) and RZ(theta)
```

Comment:
The docstring doesn't match the generated `Program`. I think you intended to swap `RZ` and `_RY` below?